### PR TITLE
[#3190] hide unaccepeted community registration request from send sur… (connects #3190)

### PR DESCRIPTION
**Pull request by singharpita**
…vey list

### CODE REVIEW

File: src/app/shared/dialogs/dialogs-list.service.ts [hunk 1]
```diff
@@ -51,7 +51,12 @@ export class DialogsListService {
   getListAndColumns(db: string, selector?: any, planetField: string = 'local') {
     selector = selector || this.defaultSelectorFunctions()[db];
     if (db === '_users') {
-      return this.couchService.findAll('_users', findDocuments({}, this.userService.userProperties))
+      return this.couchService.findAll('_users', findDocuments({
```
Comment (paulbert) [inline src/app/shared/dialogs/dialogs-list.service.ts]: @singharpita This users for communities are found in the `child_users` database on the nation.  This change will not affect the users from communities which have not been accepted.

We need to check the `communityregistrationrequests` database and filter if the community matching the `planetCode` has not been accepted when running `getListAndColumns` for `child_users`

Comment (lmmrssa) [reply src/app/shared/dialogs/dialogs-list.service.ts]: @paulbert child_users will not be synced until we approve community right. But we do send users to users database. And list is checking planetcode from both _users and child_users to show list.

**Comment by singharpita** [comment]
@paulbert there is `defaultSelectorFunctions` shall we add filter in this function? Is there any place (listview) which will require to show inactive users?

**Comment by paulbert** [comment]
@singharpita You can combine this filter with what is already there for `defaultSelectorFunctions` for `child_users`, but you will need to get the data on `communityregistrationrequests` to know what to filter each time.

**Pull request merged**